<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemEditDialog</class>
 <widget class="QDialog" name="SystemEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>358</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>668</width>
    <height>358</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>6000</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>202</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>系统设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>201</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="CPushButton" name="closeButton">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">QTabWidget{border:none;}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="system">
        <attribute name="title">
         <string>系统</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>24</number>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>12</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>     监听端口</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="portSpinBox">
                 <property name="minimumSize">
                  <size>
                   <width>62</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="styleSheet">
                  <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>录像时长(秒)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="recordTimeSpinBox">
                 <property name="minimumSize">
                  <size>
                   <width>62</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="styleSheet">
                  <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>     码流选择</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="sreamComboBox">
                 <item>
                  <property name="text">
                   <string>主码流</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>辅码流</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="systemSavePushButton">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="systemDefaultpushButton">
               <property name="text">
                <string>默认值</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>316</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="title">
         <string>存储</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>25</number>
           </property>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;仿宋&quot;;</string>
             </property>
             <property name="text">
              <string>配置平台文件保存路径（包括路线文件，抓图文件等）</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>视频文件保存路径</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="vedioDirEdit">
                   <property name="text">
                    <string>D:\l110\veido</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vedioDirButton">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>抓图文件保存路径</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="imageDirEdit">
                   <property name="text">
                    <string>D:\l110\image</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="imageDirButton">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="dirSaveButton">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dirDefaultButton">
               <property name="text">
                <string>默认值</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>274</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPushButton</class>
   <extends>QPushButton</extends>
   <header>customContro/cpushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
